sdk/python/lib/test/test_next_serialize.py

```diff
@@ -72,8 +77,24 @@ def new_resource(self, typ, name, inputs, provider, id):
         else:
             raise Exception(f"unknown resource type {typ}")
 
+
+@pulumi.output_type
+class MyOutputTypeDict(dict):
+
+    def __init__(self, values: list):
+        pulumi.set(self, "values", values)
+
+    # Property with empty body.
+    @property
+    @pulumi.getter(name="firstValue")
```

Comment:
Nit: It doesn't matter at all here, but this should really be:

```suggestion
    @pulumi.getter
```